website/src/css/custom.css

```
@@ -7,14 +7,23 @@
 
 /* You can override the default Infima variables here. */
 :root {
-    --ifm-color-primary: #25c2a0;
-    --ifm-color-primary-dark: rgb(33, 175, 144);
-    --ifm-color-primary-darker: rgb(31, 165, 136);
-    --ifm-color-primary-darkest: rgb(26, 136, 112);
-    --ifm-color-primary-light: rgb(70, 203, 174);
-    --ifm-color-primary-lighter: rgb(102, 212, 189);
-    --ifm-color-primary-lightest: rgb(146, 224, 208);
-    --ifm-code-font-size: 95%;
+    --ifm-color-primary: #00755a;
+    --ifm-color-primary-dark: #006951;
+    --ifm-color-primary-darker: #00634d;
+    --ifm-color-primary-darkest: #00523f;
+    --ifm-color-primary-light: #008163;
+    --ifm-color-primary-lighter: #008768;
+    --ifm-color-primary-lightest: #009875;
+}
+
+[data-theme='dark'] {
```

```suggestion
:root[data-theme='dark'] {
```
We only want to apply the variables to `:root`, not *every* element which may have a `data-theme='dark'` attribute. While it's unlikely to cause problems, we should stay consistent with the other, light mode, selector